exercises/dnd-character/mix.exs

```diff
@@ -0,0 +1,25 @@
+defmodule DndCharacter.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :dnd_character,
+      version: "1.1.0",
```

Comment:
This seems to match up with the canonical data version, but I think it would be better to keep the generated `"0.1.0"`.